charts/pega/templates/_pega-deployment.tpl

```diff
@@ -28,8 +28,11 @@ metadata:
     app: {{ .name }} {{/* This is intentionally always the web name because that's what we call our "app" */}}
     component: Pega
 spec:
-  # Replicas specify the number of copies for {{ .name }}
+
```

Comment:
I think that you should extract this into a named template that can be overridden in a subchart or else you may run into problems elsewhere.